### PR TITLE
[HacktoberFest] Add missing docstrings to diffusers/models

### LINKED ISSUE #5228: [HacktoberFest][Missing Docstrings] UNet2DConditionModel
This feature request is included in this years Diffusers' Hacktoberfest 👨‍💻! 

Hacktoberfest's purpose is to give upcoming open-source developers the opportunity to learn about open-source and programming by fixes simple issues (like this one).

**How to solve the issue**:

- 1. Every issue has a description further below explaining the problem. First, you should try to understand the issue.
- 2. Second, follow the [contribution guide](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md) to learn how to contribute to Diffusers.\
- 3. Claim this issue for you by opening a PR that links to this issue and writing "I am solving this issue in `<link-to-pr>`"
**Issue**

The [forward method of UNet2DConditionModel](https://github.com/huggingface/diffusers/blob/78a78515d64736469742e5081337dbcf60482750/src/diffusers/models/unet_2d_condition.py#L754) is missing a couple of docstrings, e.g. for `class_lables`, `attention_mask` ... -> can you add all the missing ones?

### PR DESCRIPTION
**Pull request by a-r-r-o-w**
# What does this PR do?

Adds missing docstrings as mentioned in the linked issue. Also adds docstrings for similar missing cases in other files not mentioned in the original issue.

Fixes #5228 

cc: @patrickvonplaten @yiyixuxu

### CODE REVIEW

File: src/diffusers/models/controlnet.py [hunk 1]
```diff
@@ -671,7 +671,12 @@ def forward(
             class_labels (`torch.Tensor`, *optional*, defaults to `None`):
                 Optional class labels for conditioning. Their embeddings will be summed with the timestep embeddings.
             timestep_cond (`torch.Tensor`, *optional*, defaults to `None`):
+                Conditional embeddings for timestep. If provided, the embeddings will be summed with the samples
+                passed through the `self.time_embedding` layer to obtain the timestep embeddings.
```
Comment (yiyixuxu) [inline src/diffusers/models/controlnet.py:675]:
```suggestion
                Additional conditional embeddings for timestep. If provided, the embeddings will be summed with the timestep embedding
                passed through the `self.time_embedding` layer to obtain the timestep embeddings.
```

**Review by yiyixuxu — APPROVED** [review]
thanks!

**Comment by yiyixuxu** [comment]
@a-r-r-o-w don't forget `make quality` !

File: src/diffusers/models/controlnet.py (hunk 1 above)
Comment (a-r-r-o-w) [reply src/diffusers/models/controlnet.py]: Added the suggestion.

**Pull request merged**